_includes/pbc/all/install-features/202304.0/install-the-inventory-management-feature.md

```diff
@@ -0,0 +1,725 @@
+     */
+    protected function getStockCollectionExpanderPlugins(): array
+    {
+        return [
+            new StockAddressStockCollectionExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return \Spryker\Zed\StockExtension\Dependency\Plugin\StockPostCreatePluginInterface[]
+     */
+    protected function getStockPostCreatePlugins(): array
+    {
+        return [
+            new StockAddressStockPostCreatePlugin(),
+        ];
+    }
+
+    /**
+     * @return \Spryker\Zed\StockExtension\Dependency\Plugin\StockPostUpdatePluginInterface[]
+     */
+    protected function getStockPostUpdatePlugins(): array
+    {
+        return [
+            new StockAddressStockPostUpdatePlugin(),
+        ];
+    }
+}
+```
+</details>
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the warehouse address management works:
+1. In the Back Office, create a warehouse.
+2. Import a warehouse address using a data import functionality.
+3. Check if the imported warehouse address exists in the `spy_stock_address` database table.
+
+{% endinfo_block %}
+
+4. Register the following plugins for warehouse allocation:
+
+| PLUGIN                                          | SPECIFICATION                             | PREREQUISITES | NAMESPACE                                                |
+|-------------------------------------------------|-------------------------------------------|---------------|----------------------------------------------------------|
+| SalesOrderWarehouseAllocationCommandPlugin      | Allocates warehouse for sales order item. | none          | Spryker\Zed\WarehouseAllocation\Communication\Plugin\Oms |
+
+
+**src/Pyz/Zed/Oms/OmsDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Oms;
+
+use Spryker\Zed\Kernel\Container;
+use Spryker\Zed\Oms\Dependency\Plugin\Command\CommandCollectionInterface;
+use Spryker\Zed\Oms\OmsDependencyProvider as SprykerOmsDependencyProvider;
+use Spryker\Zed\WarehouseAllocation\Communication\Plugin\Oms\SalesOrderWarehouseAllocationCommandPlugin;
+
+class OmsDependencyProvider extends SprykerOmsDependencyProvider
+{
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function extendCommandPlugins(Container $container): Container
+    {
+       $container->extend(self::COMMAND_PLUGINS, function (CommandCollectionInterface $commandCollection) {
+            $commandCollection->add(new SalesOrderWarehouseAllocationCommandPlugin(), 'WarehouseAllocation/WarehouseAllocate');
+
+            return $commandCollection;
+        });
+
+        return $container;
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that after the order is created, order items gain the `warehouse allocated` status.
+
+{% endinfo_block %}
+
+
+## Install an example of product and product offer warehouse allocation
+
```

Comment:
I can propose something like this: 
"Install a ProductWarehouseAllocationExample and ProductOfferWarehouseAllocationExample modules to see an example of how the warehouse allocation process for product and product offer order items can be implemented, integrated, and function." 
What do you think?

Reply:
@sakharova-yuliia, your suggestion looks like a step of the process. We need something that describes the overall sections. For example:
```suggestion
This section describes an example of how to implement and integrate the warehouse allocation process for product and product offer order items and how this process work.
```
I removed the modules because, in this sentence, it doesn't really matter what modules we need to install, we need to explain what the section stands for. The modules are listed in `### 1) Install the required modules using Composer` anyway.

Reply:
Agree, applied these changes